### DISCUSSION TITLE
Tipping?

### DISCUSSION THREAD

**Discussion by jsreynolds** [Nimdy/Dedicated_Valheim_Server_Script#139] [General]
I'd like to tip (read: Patreon) but can't seem to find a way to not sign up for something monthly. Is there a way to do a one-time tip? Im just shy of too many monthly 'services' happening all the time - hard to keep up where the $ is going.

**Comment by Nimdy** [comment]
oh I dont... its okay. I just have patreon.  Its not biggie. I am just happy to help.

**Reply by Nimdy** [reply]
I also have ETH and VET wallets


ETH: 0x691922a6b7d4005392Fb48A626B9ad17f68D66A5
VET: 0x6a2e7c13606e07d57664821a0d9920da75719c01

**Comment by Nimdy** [comment]
Finally came across this!

https://www.buymeacoffee.com/zerobandwidth

Now you dont have to do monthly stuff :)
